### ISSUE TITLE
Updating Wiki: Move Current Wiki to New Repository

### ISSUE THREAD

**Issue by blulady** [hackforla/website#3636]
### Dependency
- [x] #3634
- [x] #3635

### Overview
We need to move the current Wiki into the new repository so that we can have increased security and navigability. 

### Action Items
- [x] Read the documentation that was updated by issue #3635 
  - [x] Read the website-wiki repo's [README.md](https://github.com/hackforla/website-wiki/blob/main/README.md) 
- [x] Read the documentation in https://github.com/hackforla/admin/issues/36#issuecomment-770112547 from `Step 3. Setup your local working directory` to download the files to your local machine
- [x] upload the files to the new repos doc folder
- [x] Read the documentation from MkDocs
- [x] Let the team know you are ready to move the wiki to the website-wiki repo, so that additional edits won't be made in the old wiki
- [x] Move the current Wiki into the new Wiki

### Resources/Instructions
- https://www.mkdocs.org/
- Current wiki in website repo: https://github.com/hackforla/website/wiki
- MkDocs website-wiki repo: https://github.com/hackforla/website-wiki
- issue #3635 
- issue #3634 
- https://github.com/hackforla/admin/issues/36
- website-wiki repo's [README.md](https://github.com/hackforla/website-wiki/blob/main/README.md)

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
- Once issue #3635 is closed, did you want @vraer to start on this Large complexity issue now? Or did you want Vanessa to complete a medium complexity issue first?
- Ans: Yes, complete a medium issue first.

**Comment by JessicaLucindaCheng** [comment]
@hackforla/website-merge 
- A dev lead needs to edit this issue for more clarity
- Also, should the role labels be changed to `role: writer`? Maybe we need a new label called `role: technical writer`? Ask Bonnie
  - Edit: At 2023-03-20 Dev/PM meeting, Bonnie said it should have a `role: technical writer label`

**Comment by jdingeman** [comment]
@JessicaLucindaCheng - so this issue will only be for the technical writer, correct? This won't be assigned to a front end/back end dev?

**Comment by JessicaLucindaCheng** [comment]
@jdingeman 
> @JessicaLucindaCheng - so this issue will only be for the technical writer, correct? This won't be assigned to a front end/back end dev?

Correct, it's only for the technical writer. I changed the role labels already.

**Comment by JessicaLucindaCheng** [comment]
@t-will-gillis Since you worked on the previous issue writing up the documentation, is there anything else should be added to this issue to make it clearer?

**Comment by t-will-gillis** [comment]
@JessicaLucindaCheng These are in the `README.md` file but my takeaways from the issue: a.) All the files for the pages should be organized in the `/docs/` folder, b.) we/ someone needs to decide which of the extensions we want in the `mkdocs.yml` - maybe the [recommended configuration](https://squidfunk.github.io/mkdocs-material/setup/extensions/#recommended-configuration) and then there are adjustments in each of the extensions that I believe could come later c.) I believe when we want to deploy/ add/ or edit something we run the `ci.yml` file

**Comment by vraer** [comment]
@ExperimentsInHonesty https://github.com/hackforla/admin/issues/36#issuecomment-770112547 mentions adding the wiki as a submodule to the new repository. Does this mean the two wikis will be linked together for the long term, or is this just a temporary way to pull the files?


I've been playing around with MkDocs and some plugins locally and I think it'd be super helpful to get a clearer idea of what we want to achieve with this migration. If anyone has thoughts on these topics, I'd love to hear them:
- Main issues with current wiki
- Areas needing better org or navigation
- Long-term goals for wiki structure
- Any content categories to focus on?

Any insights or guidance on this would be awesome. Thanks!

**Comment by ExperimentsInHonesty** [comment]
@vraer Sorry about the confusion.  What I meant to ask you to do is

1. Read the documentation in https://github.com/hackforla/admin/issues/36#issuecomment-770112547 Step 3 Only to setup your local working directory and download the files to your local machine
 2. upload the files to the new repos doc folder

If you need help with this please reach out to @jdingeman or @roslynwythe 

We can answer all the rest of the questions you have after

**Comment by vraer** [comment]
@jdingeman @roslynwythe I've covered the issue tasks in the `move-current-wiki-to-new-repo-3636` branch linked to this issue. Here is a quick rundown of the changes:

- Created a Pages folder for HFLA website pages and organized GitHub Actions (GHA) and Decision Records (DR) files into separate folders (as discussed in Sunday's meeting)
  - Added separate index pages for GHA, DR, and HFLA site pages.
- Removed `Home.md` (and `_Sidebar.md`) and updated the `index.md` home page with the migrated content.
- Fixed and updated links on the `index.md` home page.

Several broken links, formatting, missing images, and outdated or duplicate info still need fixing (a lot of it seems to be research-related).

I also tried out different layouts, folder/file naming conventions, etc., in a test branch called `wiki-website-sandbox` (I didn't want to make too many changes in the main branch without knowing if there were other plans or ideas for moving forward). Note that you'll need to install the [awesome-pages plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin) in your virtual environment using `pip install mkdocs-awesome-pages-plugin` to build the site locally.
